google-cloud-dlp/.rubocop.yml

```diff
@@ -4,7 +4,7 @@ AllCops:
     - "lib/google/**/*"
     - "Rakefile"
     - "test/**/*"
-    - "acceptance/google/cloud/dlp/v2beta1/dlp_service_smoke_test.rb"
+    - "acceptance/google/cloud/dlp/v2/dlp_service_smoke_test.rb"
```

Comment:
If you want to run both tests it might be a good idea to make this `"acceptance/**/*smoke_test.rb"`